binary/pymedphys.bat

```diff
@@ -0,0 +1,18 @@
+@echo off
+
+@REM Copyright (C) 2020 PyMedPhys Contributors
+
+@REM Licensed under the Apache License, Version 2.0 (the "License");
+@REM you may not use this file except in compliance with the License.
+@REM You may obtain a copy of the License at
+
+@REM     http://www.apache.org/licenses/LICENSE-2.0
+
+@REM Unless required by applicable law or agreed to in writing, software
+@REM distributed under the License is distributed on an "AS IS" BASIS,
+@REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+@REM See the License for the specific language governing permissions and
+@REM limitations under the License.
+
+
+"%~dp0\python-embed\python" -m pymedphys %*
```

Comment:
%~dp0 is the folder in which the executing bat file resides.
%* is the set of arguments to the .bat file/script.

So it's really important that people not copy this file somewhere else.
Or if they create a link to it, that they be careful about setting the working directory.
One of the first things an MS Windows user will do is create an icon/link on their desktop, so it's important that the requirements on this be documented somewhere, and "in the .bat file itself" is a pretty good place to put a copy of that documentation

Reply:
One way the people can use this anywhere is to add the directory it is in to the users PATH, that way they can call `pymedphys dicom listen` for example in any directory, and because the bat file is in their path, it should just work.

That detail would be quite helpful in the batfile itself :). I'll do that.

Alternatively, can you think of a better way to expose the pymedphys CLI?